### PR TITLE
Move constants to config (prepare_feedin, ...)

### PR DESCRIPTION
**Pull request by SabineHaas**
See #243

### CODE REVIEW

**Review by jnnr — COMMENTED** [review]
Thanks for your work! Looks good. Some comments:
* Some of the constants names do not make a lot of sense outside of the context of the script, e.g. COL_SELECT, VAR_NAME, SCENARIO_KEY, NAME. Maybe renaming could help.
* The usage of all caps is not consistent. Some constants in the config are all caps, others not. 

Did you perform a test run?

**Comment by SabineHaas** [comment]
> Thanks for your work! Looks good. Some comments:
> 
>     * Some of the constants names do not make a lot of sense outside of the context of the script, e.g. COL_SELECT, VAR_NAME, SCENARIO_KEY, NAME. Maybe renaming could help.
> 
>     * The usage of all caps is not consistent. Some constants in the config are all caps, others not.
> 
> 
> Did you perform a test run?

No, I get an error, also when running your PR #243. I have realized just now that the tests of #243 in the CI do not fail. So there must be something wrong with my installation. 
My suggestion: take over this PR, you can use all the constants I have added.

**Comment by jnnr** [comment]
Open TODOs:

- [ ] Make it run
- [ ] use lower case letters

File: oemof_b3/tools/data_processing.py [hunk 1]
```diff
@@ -21,6 +21,12 @@
 import numpy as np
 
 
+import oemof_b3.config as config
+
+# global variables
+TS_INDEX_NAME = config.settings.general.ts_index_name
```
Comment (jnnr) [inline oemof_b3/tools/data_processing.py:27]: Don't rename

File: scripts/prepare_feedin.py [hunk 2]
```diff
@@ -32,20 +32,10 @@
...
-NUTS_DE40 = "DE40"
-RENAME_NUTS = {NUTS_DE30: "B", NUTS_DE40: "BB"}
-TS_VAR_UNIT = "None"
-TS_SOURCE = "https://www.renewables.ninja/"
-TS_COMMENT = "navigate to country Germany"
-# specific to ror time series
-REGIONS = ["BB", "B"]
-TS_SOURCE_ROR = "https://zenodo.org/record/1044463"
-TS_COMMENT_ROR = "Isolated ror availability time series from DIW data"
+TS_INDEX_NAME = config.settings.general.ts_index_name
```
Comment (jnnr) [inline scripts/prepare_feedin.py:38]: Don't rename

File: scripts/prepare_vehicle_charging_demand.py [hunk 3]
```diff
@@ -38,24 +38,27 @@
...
+)  # start charging strategy "balanced" for home profile
+HOME_END = (
+    config.settings.prepare_vehicle_charging_demand.HOME_END
+)  # end charging strategy "balanced" for home profile
+WORK_START = (
+    config.settings.prepare_vehicle_charging_demand.WORK_START
+)  # start charging strategy "balanced" for work profile
+WORK_END = (
+    config.settings.prepare_vehicle_charging_demand.WORK_END
+)  # end charging strategy "balanced" for work profile
```
[the remark this answers is not in the document]
Comment (jnnr) [reply scripts/prepare_vehicle_charging_demand.py:61]: For better readability, I will leave it as it is.

**Pull request merged**